website/pages/docs/job-specification/service.mdx

```diff
@@ -202,6 +202,12 @@ scripts.
   service. Valid options are the empty string, `passing`, `warning`, and
   `critical`.
 
+- `success_before_passing` `(int:0)` - The number of successful checks required before
```

Comment:
```suggestion
- `success_before_passing` `(int:0)` - The number of consecutive successful checks required before
```

It's consecutive right? So 0 is just one successful check?

Reply:
Ah yep, fixed